discovery-handler-modules/debug-echo-discovery-handler/src/main.rs

```diff
@@ -0,0 +1,49 @@
+use akri_debug_echo::{
+    discovery_handler::{DiscoveryHandler, DISCOVERY_PORT},
+    get_register_request,
+};
+use akri_discovery_utils::{
+    discovery::{server::run_discovery_server, DISCOVERY_HANDLER_PATH},
+    registration_client::{register, register_again},
+};
+use log::{info, trace};
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    env_logger::try_init()?;
+    info!("main - debugEcho discovery handler started");
+    // Determine whether to serve discovery handler over UDS or IP based on existence
+    // of the environment variable POD_IP.
```

Comment:
the POD_IP code looks pretty shared ... can it be moved to the discovery handler utils library?

Reply:
or maybe even the main function as a whole?

Reply:
Very good point. Fixed 1bf0c5c.